src/battle_ai_script_commands.c

```diff
@@ -533,7 +533,9 @@ static u8 ChooseMoveOrAction_Doubles(void)
                 if (i == (sBattler_AI ^ BIT_FLANK) && bestMovePointsForTarget[i] < 100)
                 {
                     bestMovePointsForTarget[i] = -1;
+                    #ifndef NONMATCHING
```

Comment:
the original purpose of these fake matches were to avoid ifdefs like this, this is just defeating the point